app/uk/gov/hmrc/accessibilitystatementfrontend/config/AppConfig.scala

```diff
@@ -32,14 +32,11 @@ case class AppConfig @Inject() (
   servicesConfig: ServicesConfig,
   visibilityParser: VisibilityParser
 ) extends Logging {
-  private val platformFrontendHost =
-    config.getOptional[String]("platform.frontend.host")
 
   private val contactFrontendHostUrl: String =
-    platformFrontendHost.getOrElse(
-      servicesConfig.getString("contact.frontend.host")
-    )
-  val reportAccessibilityProblemUrl          =
+    config.getOptional[String]("platform.frontend.host").getOrElse("")
```

Comment:
rather than getOrElse empty string we could getOrElse the baseUrl for contact-frontend from servicesConfig maybe

Reply:
@oscarduignan So this is where the complexity creeps in... `baseUrl` insists on there being a port for a URL, see here: https://github.com/hmrc/bootstrap-play/blob/main/bootstrap-common-play-30/src/main/scala/uk/gov/hmrc/play/bootstrap/config/ServicesConfig.scala#L41

I assume this is because the expected use case for ServicesConfig is for this to be used for service communication on the backend, not for generating URLs within the frontend for end users.

Reply:
@oscarduignan Having said that, I guess we WOULD only be used that use case if there was no `platform.frontend.host` defined, so we can assume it would always be OK...